.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "submodules/ios-js-support"]
+	path = submodules/ios-js-support
+	url = https://github.com/duckduckgo/ios-js-support.git
```

Comment:
How do you feel about tagging the other repo and then pointing to a specific tag here?

Reply:
Ok scrap that since we can't do that with submodules! Let's just also tag the other repo so we have a version record there.